src/NewProject/newprojectmodel.cpp

```diff
@@ -34,15 +34,15 @@ QString NewProjectModel::pageMainText(const int index) {
           "project.\r\n\r\n"
           "To create a Cwise project you will need to provide a name and a "
           "location for your project files."
-          " Next,you will specify the type of flow you'll be working with. "
-          "Finally,you will specify your project sources and choose a default "
+          " Next, you will specify the type of flow you'll be working with. "
```

Comment:
Whitespace before "Next", can be removed. I also see whitespace at the EOL which I am not sure is intentional.